modules/gschema-overrides/README.md

```diff
@@ -0,0 +1,38 @@
+# `gschema-overrides` module for startingpoint
+
+`Gschema-overrides` startingpoint module is used for including your system-setting overrides.
+If you heard about `dconf`, this is similar to it, but it's a much better & cleaner method of doing it.
```

Comment:
> This module is similar to using `dconf` configuration, but is better because it doesn't require a systemd service and supports build-time troubleshooting.

Reply:
Updated